PenguinsCanFly/Assets/Scripts/GliderModelController.cs

```diff
@@ -5,71 +5,68 @@
-        Debug.Log("SAVE:RightHandGoalZClamped:" + clampedRightGoalZ);
-        Debug.Log("SAVE:LeftHandGoalZClamped:" + clampedLeftGoalZ);
-
+        
         // TODO: find better way to merge!!! PRETTY IMPACTFUL CHANGE
         // One alternative is that you can make a line between teh two lines and run that angle instead
         // Can also require both hands have to be grabbing for it to work
         float goalRotation = (clampedRightGoalZ + clampedLeftGoalZ) / 2;
 
         // Rotate handlebar so it matches the position of hand
         Vector3 localAngles = transform.localEulerAngles;
-        
-        // Uncomment and use this instead to not rotate with pitch
-        // Vector3 rot = new Vector3(localAngles.x, localAngles.y, goalZ);  
-        
-        // TODO: this might not be smooth since it pretends the axis are independent when they are not. 
-        Vector3 rot = new Vector3(0.75f * (gliderInfo.totalPitchDegree - 90), 0, goalRotation);  
 
-        Debug.Log("SAVE:Is this number always 0:" + rot.y);
-        // 
-        // transform.localRotation = Quaternion.Slerp(transform.localRotation, Quaternion.Euler(rot), Time.deltaTime);
+        // TODO: this might not be smooth since it pretends the axis are independent when they are not. 
+        // Tilt glider up and down based on pitch!
+        Vector3 rot = new Vector3(0.75f * (gliderInfo.TotalPitchDegree - 90), 0, goalRotation);
         
-        // Below is reference code for if we want to snap handlebar to hand
         Quaternion finalLocalRotation = Quaternion.Slerp(transform.localRotation, Quaternion.Euler(rot), Time.deltaTime);
-
-        // If we want the bar to snap faster to hand movement
-        // Current setup: only snap quickly if BOTH hands are on it, otherwise, you don't have as much control
+        // Snap bar quickly if BOTH hands are on it, otherwise, you don't have as much control
         if (leftHandlebar.IsBeingHeld() && rightHandlebar.IsBeingHeld())
         {
             finalLocalRotation = Quaternion.Euler(finalLocalRotation.eulerAngles.x, finalLocalRotation.eulerAngles.y, goalRotation);
         }
-
         transform.localRotation = finalLocalRotation;
         
         // TODO: remove magic numbers
         // Change yaw based on the local rotation so glider actually turns
-        if (localAngles.z >= 5 && localAngles.z <= MAX_ROTATION_DEGREES)
+        if (localAngles.z >= 5 && localAngles.z <= MaxRotationDegrees)
         {
             gliderInfo.totalYawDegree -= localAngles.z * Time.deltaTime;
-        } else if (localAngles.z <= 355 && localAngles.z >= 360 - MAX_ROTATION_DEGREES)
+        } else if (localAngles.z <= 355 && localAngles.z >= 360 - MaxRotationDegrees)
         {
             gliderInfo.totalYawDegree += (360 - localAngles.z) * Time.deltaTime;
         }
+        
+        // Change pitch based on local rotation so you can tilt down
+        if (leftHandlebar.IsBeingHeld() && rightHandlebar.IsBeingHeld() &&
```

Comment:
Future PR but can consider the effect of just using one hand to tilt, ie. would cause you to tilt in the direction of whichever way you're pitching. In this case, the average is just evening out the two pitches, but the general case should handle it is what I think? Although I can see that this case is a little odd since you would just move your hand ever so slightly down to achieve that.

Reply:
Hm, interesting idea!

Try testing it out too and see if it feels weird, it can definitely be something we improve later.